app/components/gh-members-lab-setting.js

```diff
@@ -140,7 +148,6 @@ export default Component.extend({
             }
 
             if (key === 'currency') {
-                this.set('selectedCurrency', event.value);
```

Comment:
This wasn't working because `event.value` is a single currency value string such as `'eur'` but the `{{one-way-select}}` is expecting a currency object that matches one of the objects in the options array that was passed to it.